src/main/java/ui/listpanel/ListPanel.java

```diff
@@ -564,11 +564,12 @@ private void markAsRead() {
         }
     }
 
-    private void createNewWatchListMenuItem() {
+    /**
+     * Creates a new watch list panel containing the currently selected issue
+     */
+    private void createNewWatchListPanel() {
         Optional<GuiElement> item = listView.getSelectedItem();
-        if (!item.isPresent()) {
-            return;
-        }
+        assert item.isPresent();
```

Comment:
Yes. However, just because the issue disappeared from the view does not mean it cannot be added to the watch list. So the user will be puzzled. Perhaps we should try to prevent this happening rather than report an error. Is that possible?